doc/manpages/keys.asciidoc

```diff
@@ -629,4 +629,14 @@ map <scope> <mode> <key> <keys>
 *keys*::
 	string that lists the keys that will be executed when *key* is hit
 
-Keys are always executed within the *normal* mode.
+All mappings, except those bound to the `insert` mode, will be executed in
```

Comment:
Actually its only user and normal mode mapping that are executed in normal mode.